### ISSUE TITLE
"Expected complex" error

### ISSUE THREAD

**Issue by yurirtonin** [silx-kit/jupyterlab-h5web#115]
### Describe the bug

I am creating a dummy h5 file containing a complex array and a integer array as bellow, but I get an error message saying "Expected complex" (for the complex array) and (Expected Number) for the integer array. The same does not happen for floats; it opens normally using dtype=np.float64.

```python
import numpy as np
import h5py

file = h5py.File('test.h5','a')

name = '1'
data = np.ones((5,5),dtype=np.complex64)

file.create_dataset(name,data=data)
```

### Screenshots

<img width="985" alt="Screenshot 2023-05-16 at 11 23 55" src="https://github.com/silx-kit/jupyterlab-h5web/assets/16940168/6480b20c-5586-4ff0-ac34-0514c84ef7a6">

### Context

- OS: Ubutun 20.04.4 LTS 
- Browser: Chrome Version 112.0.5615.49 (Official Build) (x86_64)
IPython          : 7.29.0
ipykernel        : 6.5.0
ipywidgets       : 7.6.3
jupyter_client   : 7.4.9
jupyter_core     : 5.2.0
jupyter_server   : 1.24.0
jupyterlab       : 3.6.1
nbclient         : 0.5.3
nbconvert        : 7.2.9
nbformat         : 5.7.3
notebook         : 6.4.0
qtconsole        : 5.1.1
traitlets        : 5.9.0

<details><summary>Extension lists</summary>
<pre>

JupyterLab v3.6.1
/.../.local/share/jupyter/labextensions
        @jupyter-widgets/jupyterlab-manager v3.1.0 enabled OK (python, jupyterlab_widgets)

/.../apps/modules/python/3.9.2/share/jupyter/labextensions
        jupyterlab-plotly v5.1.0 enabled OK
        jupyterlab-h5web v7.0.0 enabled OK (python, jupyterlab_h5web)
        jupyter-matplotlib v0.11.2 enabled OK

config dir: /...//.local/etc/jupyter
    jupyter_server_ydoc  enabled 
    - Validating...
      X is jupyter_server_ydoc importable?
config dir: /.../modules/python/3.9.2/etc/jupyter
    jupyter_server_ydoc  enabled 
    - Validating...
      X is jupyter_server_ydoc importable?
    jupyterlab  enabled 
    - Validating...
      jupyterlab 3.6.1 OK
    jupyterlab_h5web  enabled 
    - Validating...
      jupyterlab_h5web 7.0.0 OK

</pre>
</details>

**Comment by loichuder** [comment]
It should be easy to fix: this is similar to #111 where the `dtype` query arg was not parsed. Except that it is the `flatten` arg in the case. 

Thanks for reporting the problem !

**Comment by loichuder** [comment]
@yurirtonin Please try the version 7.1.2 of the extension to see if this fixes the problem. 

I think it should also fix your _Expected Number_ error. If not, feel free to open a new issue.
